### DISCUSSION TITLE
Not a issue

### DISCUSSION THREAD

**Discussion by J-afk-bit** [ScratchAddons/ScratchAddons#2294] [Help]
This is not a issue but i need to back scratch normal so i unistall it.

**Comment by mxmou** [comment]
If it is not an issue, you don't need to report it.

**Comment by cobaltt7** [comment]
Go to chrome://extensions

Find Scratch Addons in the list

Flip the switch to disable it

**Comment by Hans5958** [comment]
Moved to Discussions.
